File: src/python/main/core/rich_json_parse.py
from .rich_json import RichJsonParser


def parse_rich_json(obj):
    """
    Parses RichJson expressions in a JSON object.

    :param obj: The JSON object (dictionary/list) to parse.
    :return: The parsed object.
    """
    print("RichJson is going to be applied...")

    return RichJsonParser().parse(obj, True)

File: src/js/main/core/RichJson_parse.js
import {RichJsonParser} from "./RichJson.js";
import {__RICH_JSON_CONFIG} from "../other/RichJsonConfiguration.js";

/**
 * Parses RichJSON expressions in JSON object.
 * @param object
 * @returns object
 */
export function parse(object) {
    return new RichJsonParser().parse(object, true);
}